rundeckapp/grails-app/views/execution/mailNotification/deprecated_status.gsp

```diff
@@ -0,0 +1,379 @@
+%{--
```

Comment:
Unless we are planning on providing a configuration option for the user to revert back to this email template, we should delete this file.

Reply:
Because we are feature flagging the new Status email, we've kept "deprecated_status", changed it's name to oldStatus and are flagging the switch in rundeck-config.properties.